package/prudynt-t/files/record-manager

```diff
@@ -10,9 +10,16 @@ terminate() {
 	/etc/init.d/S96record stop
 }
 
+getUsedSpace() {
+	if [[ $record_path = "/mnt/nfs*" ]]; then
```

Comment:
Ash is not bash. No [[ ]] here. Just [ ].

Reply:
Just this file? I had to use [[ ]] in config-record.cgi because the wildcard didnt work unless i used [[ ]] :/

Reply:
We have bash mode enabled but it is a hack. I would like our own code to stick to pure ash syntax. If something does not work without bashisms then it has to be refactored.